components/segmenter/tests/testdata/LineBreakTest.txt

```diff
@@ -1,12 +1,9 @@
-# LineBreakTest-15.0.0.txt
-# Date: 2022-02-26, 00:38:39 GMT
-# © 2022 Unicode®, Inc.
+# THIS IS NOT LineBreakTest-15.0.0.txt
```

Comment:
Are these manual changes to the test file? Please document its provenance.

Reply:
Documented (though you won’t like it :-p).